### ISSUE TITLE
Can't Install SSL Certificate

### ISSUE THREAD

**Issue by AmrAshraf** [egorovandreyrm/pcap-remote#9]
Hello,

I am trying to to install SSL Certificate and it gives me "Can't install CA certificates", "This certificate from PCAP Remote must be installed in Settings. Only install CA certificates from organizations you trust.".

Then i figured out starting from Android 11 you need to install it manually from settings.
But i can't find the right pcap remote CA to install. I also tried to Export the ssl certificate and installing it but it gives me "no certificate to install".

Can any one point me towards the CA certificate?

**Comment by aspirantis** [comment]
Hi!

I faced with the same issue. Also, generated `mitm_key.pem` include only key. So, it is not possible to convert it into `*.cer/crt`

**Comment by bostjanlaba** [comment]
I also have this issue. Can I somehow get the self-signed CA from the application and manually import it like Android wants me to?

**Comment by igrokit** [comment]
Same issue here.

**Comment by ypcyc** [comment]
Haven't tried on the real phone as I need so simulate different location, but working with emulator devices I discovered following:
1. Android 11 (api 30) doesn't allow to install that certificate
2. Android 7 (api 24) allows to install it, but in when I try to capture traffic from Chrome it give error about the certificate
3. Android 9 (api 28) works fine for me
![image](https://user-images.githubusercontent.com/1402005/150976085-41a3c3cd-5a47-4cb3-8141-75e475f02eb3.png)

**Comment by eapricejr** [comment]
Any update and/or workaround on this by chance?

**Comment by CreateDilemmas** [comment]
No, ive used a Magisk extention to push user certs to system but i cant
with just a pem, needs to be a cert.

**Comment by NicoLouis-be** [comment]
the certificate is in de code
https://github.com/egorovandreyrm/pcap-remote/tree/master/app/src/main/assets/mitm_cert

if your phone is rooted and you know how to use ADB then follow these instructions from step 2 point 2 for the cert.pem file of the above link
https://secabit.medium.com/how-to-configure-burp-proxy-with-an-android-emulator-31b483237053

don't know if the output of the openssl commands are the same for everyone but for me the file name was 0595058e.0

did it yesterday and it works

**Comment by emanuele-f** [comment]
Since there is no activity in this repo since 2020, I suggest you to check out my open source project [PCAPdroid](https://github.com/emanuele-f/PCAPdroid) which allows you to capture traffic, perform TLS decryption and more. Join [the telegram group](https://t.me/PCAPdroid) to get in touch
